runtime/devnet/Cargo.toml

```diff
@@ -258,8 +258,10 @@ try-runtime = [
 # generate the metadata hash and then a second time with the
 # `RUNTIME_METADATA_HASH` environment variable set for the `CheckMetadataHash`
 # extension.
-metadata-hash = [ "substrate-wasm-builder/metadata-hash" ]
+metadata-hash = ["substrate-wasm-builder/metadata-hash"]
 
 # A convenience feature for enabling things when doing a build
 # for an on-chain release.
-on-chain-release-build = [ "metadata-hash" ]
+on-chain-release-build = ["metadata-hash"]
+
+drink = []
```

Comment:
This is causing issues in CI

Reply:
taplo and zepter want different formatting here. 

We should see if there is a way to put something in there and bypass the ci checks. 

Already checked whether there were changes that we could make to the taplo.toml file but unsuccesful.

Making changes to either one of those repos sounds a waste of time.

Reply:
Fixed